dashboard/app/models/course_offering.rb

```diff
@@ -188,9 +188,21 @@ def any_version_is_in_published_state?
 
   def self.all_course_offerings
     if should_cache?
-      @@course_offerings ||= CourseOffering.all.includes(course_versions: :content_root)
+      @@course_offerings ||= CourseOffering.all.includes(
+        course_versions: {
+          content_root: {
+            default_unit_group_units: {}
+          }
+        }
+      )
     else
-      CourseOffering.all.includes(course_versions: :content_root)
+      CourseOffering.all.includes(
+        course_versions: {
+          content_root: {
+            default_unit_group_units: {}
+          }
+        }
+      )
```

Comment:
Thank you for considering the development case. on my local machine, this improves time from page reload until Owned Sections table appears (with course column populated) from 30s to 23s 👍 

I didn't benchmark other routes which use `all_course_offerings` (which might not use `unit_group_unit`), but the homepage likely gets the most use, so I feel ok about that.